doc/contributing/releases.md

```diff
@@ -545,6 +546,24 @@ with the Build Working Group.
 
 </details>
 
+#### 7.1 Updating the release _(optional)_
+
+Sometimes a release might be deferred to the subsequent day due to several
+conditions:
+
+* Unstable CI
+* Bad commit, requiring a git bisect and updating the proposal
```

Comment:
I think it'd be useful to have the Git steps documented in this section for when we need to remove or adjust commits in the release. As a draft suggestion, I think including steps like this would be useful:

Remove or adjust problematic commit(s):

```
# update staging
git checkout staging
git rebase -i ...  # rebase to drop or edit the bad commit(s)
git push -f upstream staging
# rebase proposal against the updated staging branch
git checkout proposal
git rebase upstream/staging
```

Reminder to re-run `changelog-maker` and update the changelog to include the new/changed Git SHAs. (Easily forgotten).

Reply:
Just updated

Reply:
Actually, it's missing a fix. In case of dropping a commit we need to reset the branch instead of rebasing

```
git reset --hard upstream/staging
```

Reply:
Done!

Reply:
@rafaelGSS - I never use the tmp branch method in your latest addition: 

```console
$ git checkout v1.2.3-proposal
$ git checkout -b v1.2.3-proposal-tmp
$ git checkout v1.2.3-proposal

$ git reset --hard upstream/v1.x-staging
$ git cherry-pick v1.2.3-proposal-tmp
```

I always rebase the proposal branch against `upstream/staging` after staging has been updated - without the need for a tmp branch. When is the tmp branch method necessary?